portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,28 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function randomizeImage() {
+  // The images directory contains 13 images, so generate a random index between
+  // 1 and 13.
+  const imageIndex = Math.floor(Math.random() * 8) + 1;
+  const imgUrl = 'images/dino_' + imageIndex + '.jpg';
```

Comment:
does this work even though imageIndex is type int? Should you maybe convert to string?

Reply:
Thanks for catching that. It worked, i guess because of javascript type conversion. I am going to change it.